packages/release-mastermind/src/contexts/schedule.ts

```diff
@@ -0,0 +1,138 @@
+/** @format */
+
+import * as core from "@actions/core"
+import { Context } from "@actions/github/lib/context"
+import { LoggingDataClass, LoggingLevels } from "@videndum/utilities"
+import { log } from ".."
+import { Config, Runners, SharedConfig } from "../action"
+import { CurContext, ScheduleContext } from "../conditions"
+import { Utils } from "../utils"
+import { Contexts } from "./methods"
+
+/**
+ * The schedule configuration
+ */
+export type ScheduleConfig = SharedConfig
+
+/**
+ * The schedule class.
+ * @private
+ */
+export class Schedule extends Contexts {
+	context: ScheduleContext
+	ctx: ScheduleContext
+	config: ScheduleConfig
+
+	constructor(
+		util: Utils,
+		runners: Runners,
+		configs: Config,
+		curContext: CurContext,
+		dryRun: boolean
+	) {
+		if (curContext.type !== "schedule")
+			throw new LoggingDataClass(
+				LoggingLevels.error,
+				"Cannot construct without schedule context"
+			)
+		super(util, runners, configs, curContext, dryRun)
+		this.context = curContext.context
+		this.ctx = curContext.context
+		if (!configs.schedule)
+			throw new LoggingDataClass(
+				LoggingLevels.error,
+				"Cannot start without config"
+			)
+		this.config = configs.schedule
+	}
+
+	/**
+	 * Parse the Schedule Context
+	 * @author TGTGamer
+	 * @since 1.0.0
+	 */
+
+	static async parse(context: Context): Promise<ScheduleContext | undefined> {
+		return {
+			ref: context.ref,
+			sha: context.sha,
+			action: context.payload.action as string
+		}
+	}
+
+	async run(attempt?: number) {
+		if (!this.config)
+			throw new LoggingDataClass(
+				LoggingLevels.warn,
+				"Cannot start without config"
+			)
+		if (!attempt) {
+			attempt = 1
+			core.startGroup("Schedule Actions")
+		}
+		const seconds = attempt * 10
+		try {
+			const issues = await this.util.api.issues.list({})
+			issues.forEach(async (issue) => {
+				const labels = await this.util.parsingData
+					.labels(issue.labels)
+					.catch((err) => {
+						log(LoggingLevels.error, `Error thrown while parsing labels: `, err)
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Videndum/action-masterminds/pullRequest?prid=7872005)
```suggestion
            log(LoggingLevels.error, "Error thrown while parsing labels: ", err)
```